cypress/support/utils.ts

```diff
@@ -20,6 +23,8 @@ export const getAuthToken = () =>
     .its("auth")
     .its("token")
     .then(token => `Token ${token}`);
-export const getDictionaryId = () => Cypress.env("dictionaryId");
-export const setDictionaryId = (dictionaryId: string) => Cypress.env("dictionaryId", dictionaryId);
-export const getUser = () => Cypress.env("USER") || "admin";
+export const getDictionaryId = () => dictionaryId;
```

Comment:
I don't think you need to change these we are actually moving towards have environment variables instead of local variables, so maintain using the `Cypress.env` even for the organisationId.